packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -519,6 +537,7 @@ export function terminateAllRunningTeamsfxTasks(): void {
   }
   allRunningTeamsfxTasks.clear();
   BaseTunnelTaskTerminal.stopAll();
+  void deleteAad();
```

Comment:
This looks like an async background action. Would it be a corner case if I stop then re-debug in short time?